src/utils/siteLaunchUtils.ts

```diff
@@ -9,3 +9,14 @@ export const shouldUseSiteLaunchFeature = (siteName: string): boolean => {
   }
   return WHITELISTED_REPOS ? WHITELISTED_REPOS.includes(siteName) : false
 }
+
+const isAdminUser = (email: string): boolean => {
+  return email.endsWith(".gov.sg")
+}
+
```

Comment:
Are we only allowing `.gov.sg` emails to launch sites? We also have healthcare orgs and other stuff which are considered site owners, but don't have `.gov.sg` emails. Is using the site member information from the backend insufficient?

Reply:
Suggest exposing an endpoint on the backend tgt with a custom hook so that we can get all non-null expiry values for whitelist and use that for owners. 

For now, this is fine **if** our is only gov us rs.

Reply:
Other option is we manually sync but v prone to error